### PR TITLE
Adding tutorial for converting data to COCO format.

### PR DESCRIPTION
**Pull request by yongxinw**
*Issue #, if available:*

*Description of changes:*
Added a tutorial for converting data to COCO format for AutoMM detection.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/tutorials/multimodal/convert_data_to_coco_format.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+
+
+
+
+
+
+The following is an example of one sample annotated with COCO format
+
+## Converting Pascal VOC dataset to MSCOCO format
```
Comment (FANGAreNotGnu) [inline docs/tutorials/multimodal/convert_data_to_coco_format.md:133]: Should be "Covert VOC format dataset to COCO"

File: docs/tutorials/multimodal/convert_data_to_coco_format.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+The script can be found in the following directory:
+```
+./examples/automm/object_detection/voc2coco.py
+```
+
+Assuming your VOC dataset has the following structure
+```
+<path_to_VOCdevkit>/
+    VOC2007/
```
Comment (FANGAreNotGnu) [inline docs/tutorials/multimodal/convert_data_to_coco_format.md:145]: The VOC format structure should be the structure under VOC2007

File: docs/tutorials/multimodal/convert_data_to_coco_format.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+```
+
+#### 1. Go to the script folder:
+```
+cd ./examples/automm/object_detection/
+```
+
+#### 2. Run the following command:
+```
+python voc2coco.py <path_to_VOCdevkit> \
```
Comment (FANGAreNotGnu) [inline docs/tutorials/multimodal/convert_data_to_coco_format.md:159]: Change to use the voc2coco.py in our example.

File: docs/tutorials/multimodal/convert_data_to_coco_format.md [hunk 4]
```diff
@@ -0,0 +1,169 @@
+# Convert Data to MSCOCO Format
```
Comment (FANGAreNotGnu) [inline docs/tutorials/multimodal/convert_data_to_coco_format.md:1]: We shall be consistent in the naming (among tutorials). I think COCO might be better.

File: docs/tutorials/multimodal/convert_data_to_coco_format.md [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+```
+For the sole purpose of running AutoMM, the fields ``"info"`` and ``"licenses"`` are optional. 
+``"images"``, ``"categories"``, and ``"annotations"`` are required.
+
+
+
+```json
+{
+  "info": {...},
+"licenses": [
```
Comment (bryanyzhu) [inline docs/tutorials/multimodal/convert_data_to_coco_format.md:91]: The indentation here seems not right, need 4 more spaces.

File: docs/tutorials/multimodal/convert_data_to_coco_format.md [hunk 6]
```diff
@@ -0,0 +1,169 @@
+# Convert Data to MSCOCO Format
+
+:label:`convert_data_to_coco_format`
```
Comment (FANGAreNotGnu) [inline docs/tutorials/multimodal/convert_data_to_coco_format.md:3]: Here I suggest using sec_automm_detection_convert_to_coco.

File: docs/tutorials/multimodal/convert_data_to_coco_format.md (hunk 2 above)
Comment (yongxinw) [reply docs/tutorials/multimodal/convert_data_to_coco_format.md]: Addressed in ce884dba67c46572c8b5c699980d4cfebf55cdd0

File: docs/tutorials/multimodal/convert_data_to_coco_format.md (hunk 3 above)
Comment (yongxinw) [reply docs/tutorials/multimodal/convert_data_to_coco_format.md]: Addressed in ce884dba67c46572c8b5c699980d4cfebf55cdd0

File: docs/tutorials/multimodal/convert_data_to_coco_format.md (hunk 4 above)
Comment (yongxinw) [reply docs/tutorials/multimodal/convert_data_to_coco_format.md]: Addressed in ce884dba67c46572c8b5c699980d4cfebf55cdd0

File: docs/tutorials/multimodal/convert_data_to_coco_format.md (hunk 5 above)
Comment (yongxinw) [reply docs/tutorials/multimodal/convert_data_to_coco_format.md]: Addressed in ce884dba67c46572c8b5c699980d4cfebf55cdd0

File: docs/tutorials/multimodal/convert_data_to_coco_format.md (hunk 6 above)
Comment (yongxinw) [reply docs/tutorials/multimodal/convert_data_to_coco_format.md]: Addressed in ce884dba67c46572c8b5c699980d4cfebf55cdd0

**Pull request closed without merging**